src/status_im/contexts/wallet/events.cljs

```diff
@@ -386,3 +389,37 @@
             {:title   title
              :subject title
              :message content})]]}))
+
+(rf/reg-event-fx
+ :wallet/blockchain-status-changed
+ (fn [{:keys [db]} [{:keys [message]}]]
+   (let [chains                  (-> (transforms/json->clj message)
+                                     (update-keys (comp utils.number/parse-int name)))
+         down-chains             (-> (select-keys chains
+                                                  (for [[k v] chains :when (= v "down")] k))
+                                     keys)
+         test-networks-enabled?  (get-in db [:profile/profile :test-networks-enabled?])
+         filter-fn               (fn [chain-id]
+                                   (let [mainnet-chain? (contains? constants/mainnet-chain-ids chain-id)]
+                                     (if test-networks-enabled?
+                                       mainnet-chain?
+                                       (not mainnet-chain?))))
+         chains-filtered-by-mode (remove filter-fn down-chains)
```

Comment:
To prevent showing the toast if a testnet chain is down and the user is not in testnet mode.